README.md

```diff
@@ -34,23 +34,27 @@ DeepCode uses symbolic AI to process hundreds of millions of commits in open sou
 
 ## Our AI provides explanation behind found bugs
 
-In order to show the detailed explanation of a potential bug, we introduced a new AI technique called _Ontology_. With Ontology, we’ve integrated the capability to present logical argumentation used by the DeepCode engine. 
+In order to show the detailed explanation of a potential bug, we introduced a new AI technique called _Ontology_. With Ontology, we’ve integrated the capability to present logical argumentation used by the DeepCode engine. If you want to learn more about the technologies behind DeepCode, make sure to visit our website [DeepCode.AI](https://deepcode.ai) and the resources listed.
 
 ## Supported languages
 
-JavaScript, TypeScript, Java, C/C++, and Python are currently supported.
+JavaScript, TypeScript, Java, C/C++ (beta), and Python are currently supported. We also provide specific coverage for VUE and REACT.
 
-# Installation
+## Video on how to install and use the extension
+
+- We've prepared a 2 minutes video on how to install and use the extension. Head over to YouTube to quickly get the extension up and running:
```

Comment:
@CU-0xff btw video is a bit longer than 2 minutes ;)

Reply:
True :-) I changed to short :-) In the video comments I provided a clickable ToC - so it should be easy to navigate.